Partition (WBP)/Functions/VI0049D.VTI0049B.fun.yml

```diff
@@ -104,9 +104,9 @@
     Define Field(#Lf_j) Reffld(#std_Count)
     
     #DB_Type := 'DBTYPE'
-    #Literal := 'Literal'
-    #Bind := 'Bind'
-    #Lansa := 'LANSA'
+    #Literal := ' ' + 'Literal'
```

Comment:
I never saw this code with spaces. These spaces are not required.

Reply:
[PerformanceMetricsTable.txt](https://github.com/lansa/db-regression/files/14107563/PerformanceMetricsTable.txt)
[PerformanceMetricsLog.txt](https://github.com/lansa/db-regression/files/14107564/PerformanceMetricsLog.txt)
Can you please review these 2 files ?

Reply:
You will need to read the current contents of the file using TRANSFORM_FILE, like in VI0049D, into #w_lLansaExecutionList.

Do it before this line because #LANSA will be overwritten by this process:
```
#LANSA := ' ' + 'LANSA'
```

Reply:
Should I append #w_lLansaExecutionList to the log file for every database run? The logfile is getting deleted for every MSSQLS run and is getting overwritten.